src/data/markdown/docs/03 cloud/01 Creating and running a test/02 Cloud tests from the CLI/200 Cloud execution reference.md

```diff
@@ -50,6 +52,20 @@ If you want to activate it, contact our CS team at support@k6.io.
 
 </Blockquote>
 
+<Blockquote mod="note" title="">
+
+The `drop_metrics` and `drop_tags` options are for advanced users that are following good practices
+when writing their tests but are still hitting the "Too many time series" alert, resulting in an
```

Comment:
```suggestion
The `drop_metrics` and `drop_tags` options are for advanced users that are following good practices
when writing their tests but are still hitting the "Too many time series" alert
```

I don't think we should say advanced or "good practices" since these are subjective terms.

Can you provide an example of when a user might use these options and why they might benefit? Like what's a good reason to generate so many metrics, and what benefit comes from dropping them? I think we can reframe this as an example of a rare case for certain circumstances, rather than ask readers to decide whether they're advanced or not.

Reply:
We had some users that even following these tips, still hit the limits:

<img width="668" alt="Screenshot 2023-01-31 at 15 25 31" src="https://user-images.githubusercontent.com/23944/215787033-dc2718ee-bd43-4ae3-9103-1ae03872fa70.png">

What the screenshot explains is what I meant by "good practices".

Reply:
@rafa-munoz it might be good to add this info to the Performance Insights tooltip in the app as well if we haven't already.

Reply:
I think for now, we can at least link this from the Performance insights page  (and back)

Reply:
Also it appears we need to document the "Too many time series" performance metric :laughing:

Reply:
@Vmihajlovic I will add a mention back in the tooltip when we have this merged